### PR TITLE
make fish not drop after restart

### PR DESCRIPTION
**Pull request by tanmayc05**

### CODE REVIEW

File: scripts/controls.js [hunk 1]
```diff
@@ -60,15 +60,16 @@ function initializePointsText() {
 const pointsText = initializePointsText();
 
 export function addFishToDrop(position) {
-    const randomFishClass =
-        fishClasses[Math.floor(Math.random() * fishClasses.length)];
+    console.log("addFishToDrop");
```
Comment (ankitkhatri1) [inline scripts/controls.js]: comment out so console doesnt get spammed

File: scripts/controls.js [hunk 2]
```diff
@@ -138,28 +139,28 @@ export function handleMouseMove(event) {
...
-    }
-    // Right arrow key
-    else if (keyCode === 39) {
-        moveFish("right");
-    }
-    // Down arrow key
-    else if (keyCode === 40 || keyCode == 32) {
-        dropFish();
+        // Left arrow key
+        if (keyCode === 37) {
```
Comment (ankitkhatri1) [inline scripts/controls.js:146]: what the logic behind this

File: scripts/controls.js [hunk 3]
```diff
@@ -138,28 +139,28 @@ export function handleMouseMove(event) {
...
+        else if (keyCode === 40 || keyCode == 32) {
+            dropFish();
+        }
     }
 }
 
 
 export function dropFish(event) {
     if (followFish && allowInput && !isGameOver) {
+        console.log("dropFish");
```
Comment (ankitkhatri1) [inline scripts/controls.js]: common out

Comment (ankitkhatri1) [reply scripts/controls.js]: comment*

**Review by ankitkhatri1 — CHANGES_REQUESTED** [review]

File: scripts/controls.js (hunk 2 above)
Comment (tanmayc05) [reply scripts/controls.js:146]: the gameOver boolean makes it so that the fish cant move around or anything when game over screen shows up

**Pull request merged**